task_manager/task_manager.cc

```diff
@@ -5,7 +5,7 @@
 /*
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-#include "task_manager.hh"
+#include "test_module.hh"
```

Comment:
please keep both includes.
in release build mode the test_module header file should provide nothing
and then this source file should be self sufficient